src/parser/asts/variables.c

```
@@ -1,72 +1,85 @@
-	node->right = parseVariableValue(result, index + 3);
+    mod->name = result.tokens[index].value;
 
-	if(node->right != NULL) node->endingIndex = node->right->endingIndex;
-	else node->endingIndex = index + 2;
+    void* value = parseValueGroup(result, index + 2);
 
-	return node;
-}
+    if(value == NULL) {
+        printf("Error: Couldn't parse variable value group in redefinition!\n");
+        return NULL;
```

cppcheck (2.10-2)

[error] Memory leak

(memleak)